File: problems/0009_palindrome-number/to-string/solution.go
import "strconv"

func isPalindrome(x int) bool {
    if x < 0 {
        return false
    } else if x < 10 {
        return true
    }
    s := strconv.Itoa(x)
    i, j := 0, len(s) - 1
    for i < j {
        if s[i] != s[j] {
            return false
        }
        i++
        j--
    }
    return true
}

File: problems/0009_palindrome-number/to-string/solution.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function (x) {
    'use strict'
    if (x < 0)
        return false
    else if (x < 10)
        return true
    const s = String(x)
    for (let i = 0, j = s.length - 1; i < j; ++i, --j)
        if (s[i] !== s[j])
            return false
    return true
};
